lusidtools/cocoon/cocoon.py

```diff
@@ -1551,15 +1563,16 @@ def load_from_data_frame(
         data_frame_columns, "DataFrame Columns"
     )
 
-    Validator(property_columns, "property_columns").check_subset_of_list(
+    source_columns = [column["source"] for column in property_columns]
```

Comment:
What happens if the value in column["source"] is not a string?

Reply:
Good point, will add a test for that scenario

Reply:
Okay nice